cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/readJsonFile/instancesConfigurationsJsonFile/AwsInstancesConfigurations.java

```diff
@@ -0,0 +1,35 @@
+package org.cloudsimplus.sla.readJsonFile.instancesConfigurationsJsonFile;
```

Comment:
This class should be removed. You just need a InstancesAws class (that must be renamed, see the comment below).